runelite-client/src/main/java/net/runelite/client/plugins/timers/PvpWorldType.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2018, Jordan Atwood <jordan.atwood423@gmail.com>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.timers;
+
+import net.runelite.api.WorldType;
+
+enum PvpWorldType
```

Comment:
I would somehow reduce the need to basically have this structure what is really similar to one in XP tracker.

Reply:
Mentioned in discord, but I can't think of a good way to do this without completely rewriting WorldType, which seems like overengineering it.

Reply:
```java
static list <WorldType> DIFF = { DMM, SDMM, TOURNAMENT } 
static list <WorldType> PVP = { PVP }.join(DIFF)
boolean WorldType#isDifferentAccountWorld(WorldType other)
{
return 
  // hopping to different acc world
  (DIFF.contains(other) && !DIFF.contains(this)) || 
  // hopping from different acc world
  (!DIFF.contains(other) && DIFF.contains(this))
}

boolean WorldType#isPvp()
{
  return PVP.contains(this)
}
```

something like this probably should be enough

Reply:
It was a little trickier than that (hopping between `DIFF` types also required clearing tbs), but I went with that approach.